bin/resetdb.js

```diff
@@ -38,17 +36,17 @@ const runSeedFiles = async () => {
 
 const runResetDB = async () => {
```

Comment:
I'm a bit confused here, as we _do_ have an `async` function, yet as far as I know, we don't teach `async` to students.  So which is it?

Reply:
We were using the `pg-native` package in this file because it has a sync option for running queries. However, a number of students had issues with `pg-native` over the years so we converted this file to use the `node-postgres` library instead. `node-postgres` doesn't have a sync option so async/await was used as a work-around to make sure the queries run in order.

This file is not be touched by the students. It's expected that they consume it through the script (`npm run db:reset`).